src/internal_events/common.rs

```diff
@@ -92,6 +92,27 @@ impl InternalEvent for AwsBytesSent {
     }
 }
 
+#[cfg(feature = "aws-core")]
+pub struct AwsSdkBytesSent {
+    pub byte_size: usize,
+    pub region: Option<aws_types::region::Region>,
+}
+
+#[cfg(feature = "aws-core")]
+impl InternalEvent for AwsSdkBytesSent {
+    fn emit_logs(&self) {
+        trace!(message = "Bytes sent.", byte_size = %self.byte_size, region = ?self.region);
+    }
+
+    fn emit_metrics(&self) {
+        counter!(
+            "component_sent_bytes_total", self.byte_size as u64,
+            "protocol" => "https",
```

Comment:
This is likely to be fairly uncommon, but I think the protocol might actually be `http` if they configure a custom `endpoint`?

Reply:
This was just copied from `AwsBytesSent` (which was rusoto specific), which hard codes it. Is the distinction between them important? I'm leaning towards making a new issue to migrate this after the SDK migration is done if it's important (so it also changes the rusoto version of this event)